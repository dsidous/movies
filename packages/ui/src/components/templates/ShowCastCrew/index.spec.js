import React from 'react';
import { shallow } from 'enzyme';
import CastCrew from '.';

const mockProps = {
  show: {
    name: 'name',
    first_air_date: '01.01.2010',
    poster_path: 'path/to/image',
    id: 1,
  },
  loading: false,
};

describe('Templates/ShowCastCrew', () => {
  it('should render as expected', () => {
    const wrapper = shallow(<CastCrew {...mockProps} />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should render a spinner when loading expected', () => {
    const wrapper = shallow(<CastCrew {...mockProps} loading />);

    expect(wrapper).toMatchSnapshot();
  });
});
