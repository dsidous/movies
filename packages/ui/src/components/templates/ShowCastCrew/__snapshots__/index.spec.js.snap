// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Templates/ShowCastCrew should render a spinner when loading expected 1`] = `ShallowWrapper {}`;

exports[`Templates/ShowCastCrew should render as expected 1`] = `ShallowWrapper {}`;
